package.json

```diff
@@ -97,6 +97,6 @@
     "zod": "^3.19.1"
   },
   "peerDependencies": {
-    "jotai": ">=1.11.0"
+    "jotai": ">=2"
```

Comment:
Now I'm fine with this change, because it's been a while since v2 is released.
Just curious if it doesn't work with v1.11.0. Which API is missing?

Reply:
setSelf? Also we are using private Debug for dev tool support

Reply:
I see. Got it!